### PR TITLE
bug fix for some facets

### PR DESCRIPTION
**Pull request by andreacosolo**

### CODE REVIEW

File: src/encoded/schemas/antibody.json [hunk 1]
```diff
@@ -80,13 +80,19 @@
...
-            "title" : "Vendor"
+        "antibody_vendor.display_title" : {
+            "title": "Vendor"
+        },
+        "antibody_target.display_title": {
+            "title": "Target"
         }
     },
     "columns": {
         "antibody_vendor": {
```
Comment (aschroed) [inline src/encoded/schemas/antibody.json:91]: Is not a similar change required here?

**Review by aschroed — APPROVED** [review]
Have these never worked as expected or did something change due to embedding changes?

Also one in line comment.

File: src/encoded/schemas/antibody.json (hunk 1 above)
Comment (andreacosolo) [reply src/encoded/schemas/antibody.json:91]: Surprisingly it works without that change (not sure why though).

Comment (andreacosolo) [reply src/encoded/schemas/antibody.json:91]: I suspect these have never worked, but it's not something used too often to really notice.

Comment (aschroed) [reply src/encoded/schemas/antibody.json:91]: I had thought that the default when a property was not included was to use display_title and apparently that works for columns but I guess not for facets.

**Pull request merged**